packages/prover/scripts/generate_fixtures.ts

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-console */
+import {writeFile, mkdir} from "node:fs/promises";
+import path from "node:path";
+import url from "node:url";
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Is this disable necessary?

Reply:
It requires to specify `axios` and few other packages as `dependencies`, while we just need it for development. So why had to disable this.